torch/_inductor/fx_passes/post_grad.py

```diff
@@ -106,7 +121,7 @@ def post_grad_passes(gm: torch.fx.GraphModule, is_inference: bool):
             inductor_before_change = save_inductor_dict(
                 [pattern_matcher_pass.pass_name]
             )
-            pattern_matcher_pass.apply(gm.graph)  # type: ignore[arg-type]
+            apply_pass(lambda: pattern_matcher_pass.apply(gm.graph))  # type: ignore[arg-type]
```

Comment:
name?

Reply:
Many of the passes below seem like we could relatively easily give names to them

Reply:
Will udpate pr so that it is just done in GraphTransformObserver

Reply:
actually, want to land, will do in follo wup